test/common/testlib.py

```diff
@@ -1155,10 +1170,15 @@ def assert_pixels(self, selector: str, key: str,
         for layout in self.layouts:
             if layout["name"] not in skip_layouts:
                 self.set_layout(layout["name"])
+                if "rtl" in self.current_layout["name"]:
+                    self._set_direction("rtl")
                 self.assert_pixels_in_current_layout(selector, key, ignore=ignore,
                                                      scroll_into_view=scroll_into_view,
                                                      wait_animations=wait_animations,
                                                      wait_delay=wait_delay)
+
+                if "rtl" in self.current_layout["name"]:
+                    self._set_direction("ltr")
```

Comment:
We should do this only once at the end, but this only becomes relevant once we introduce a second RTL scenario. We can do this later.